src/rust/ide/json-rpc/src/error.rs

```diff
@@ -9,7 +9,6 @@ use crate::messages::Response;
 use futures::channel::oneshot::Canceled;
 
 
-
```

Comment:
We should not remove this. Before "big" section headers we require three empty lines.